### PR TITLE
glass: Add consumed storage by service dashboard widget

### PR DESCRIPTION
**Pull request by votdev**
Bildschirmfoto vom 2021-03-29 16-27-38

Fixes: https://github.com/aquarist-labs/aquarium/issues/344

Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

**Comment by jecluis** [comment]
Looks pretty snazzy. +1 for the looks ;)

File: src/glass/src/app/pages/dashboard-page/dashboard-page.component.ts [hunk 1]
```diff
@@ -39,6 +39,10 @@ export class DashboardPageComponent implements OnInit {
     {
       id: 'hosts',
       title: TEXT('Hosts')
+    },
+    {
+      id: 'services-capacity',
```
Comment (s0nea) [inline src/glass/src/app/pages/dashboard-page/dashboard-page.component.ts:44]: I'm wondering if it should be shown by default. So... I mean to add `services-capacity` to `enabled`:

```
enabled: string[] = ['health', 'capacity', 'services', 'hosts', 'services-capacity'];
```

File: src/glass/src/app/core/dashboard/widgets/services-capacity-dashboard-widget/services-capacity-dashboard-widget.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<glass-widget [loadData]="loadData.bind(this)"
```
Comment (s0nea) [inline src/glass/src/app/core/dashboard/widgets/services-capacity-dashboard-widget/services-capacity-dashboard-widget.component.html:1]: Unfortunately another Chrome (Version 89.0.4389.82) issue. It doesn't happen on Firefox.

screen-capture

Comment (Devp00l) [reply src/glass/src/app/core/dashboard/widgets/services-capacity-dashboard-widget/services-capacity-dashboard-widget.component.html:1]: I don't see that happen in chrome or chromium on Ubuntu 18.04. Maybe distro related?

File: src/glass/src/app/pages/dashboard-page/dashboard-page.component.ts (hunk 1 above)
Comment (Devp00l) [reply src/glass/src/app/pages/dashboard-page/dashboard-page.component.ts:44]: Sounds like a good idea - I'll fix that afterwards.

**Review by Devp00l — APPROVED** [review]
LGTM

**Pull request merged**